docs/help.md

```diff
@@ -244,3 +244,28 @@ in the cache, which is then treated as a fragment match.
 
 When you pass an introspected schema to the cache, this warning will never be displayed
 as the cache can then do deterministic fragment matching using schema information.
+
+## (17) Invalid type <a id="17"></a>
```

Comment:
Looks good :+1:

My two cents - I think having populate errors documented alongside graphcache might be confusing (separate exchanges, unrelated errors)

Reply:
I think it's fine-ish. I expect that some may become shared as we start sharing some utilities (if that ever becomes a thing) and I expect them to be looked up (via links in the error/log output) individually, so the user will only look at the linked section that they're currently interested in